src/DynamoRevit/DynamoRevit.cs

```diff
@@ -289,44 +290,9 @@ public Result ExecuteCommand(DynamoRevitCommandData commandData)
 
                 UpdateSystemPathForProcess();
 
-                // create core data models
-                RevitDynamoModel = InitializeCoreModel(extCommandData);
-                RevitDynamoModel.UpdateManager.RegisterExternalApplicationProcessId(Process.GetCurrentProcess().Id);
-                RevitDynamoModel.Logger.Log("SYSTEM", string.Format("Environment Path:{0}", Environment.GetEnvironmentVariable("PATH")));
-
-                // handle initialization steps after RevitDynamoModel is created.
-                RevitDynamoModel.HandlePostInitialization();
-                ModelState = RevitDynamoModelState.StartedUIless;
-
-                // show the window
-                if (CheckJournalForKey(extCommandData, JournalKeys.ShowUiKey, true))
-                {
-                    RevitDynamoViewModel = InitializeCoreViewModel(RevitDynamoModel);
-
-                    // Let the host (e.g. Revit) control the rendering mode
-                    var save = RenderOptions.ProcessRenderMode;
-                    InitializeCoreView(extCommandData).Show();
```

Comment:
Notice we no longer show the Dynamo window directly here

Reply:
I am curious about the system test, before looks open the .dyn file and run it then get nodes result.
will this affect these regression?

Reply:
![image](https://user-images.githubusercontent.com/62596335/203462508-1c4ee9a0-9db4-4c85-984c-257f48bb1797.png)
looks just open the file then RunCurrentModel, and compare the node's result

Reply:
Then this kind of tests should not be impacted